### ISSUE TITLE
Can't run docker-compose on latest master

### ISSUE THREAD

**Issue by benjick** [DivanteLtd/vue-storefront#3018]
## Current behavior
Not possible to start a clean install from `master` branch in docker with `docker-compose up`

```
app_1  | ERROR in /var/www/core/lib/multistore.ts
app_1  | ./core/lib/multistore.ts
app_1  | [tsl] ERROR in /var/www/core/lib/multistore.ts(137,41)
app_1  |       TS2345: Argument of type 'object' is not assignable to parameter of type '{ [key: string]: unknown; }'.
app_1  |   Index signature is missing in type '{}'.
app_1  | 
app_1  | ERROR in /var/www/core/lib/search/adapter/api/searchAdapter.ts
app_1  | [tsl] ERROR in /var/www/core/lib/search/adapter/api/searchAdapter.ts(74,45)
app_1  |       TS2345: Argument of type 'HttpQuery' is not assignable to parameter of type '{ [key: string]: unknown; }'.
app_1  |   Index signature is missing in type 'HttpQuery'.
```

## Expected behavior
The codebase should compile

## Steps to reproduce the issue
* Clone `vue-storefront` repo
* Run installer
* `docker-compose up`

## Repository
https://github.com/DivanteLtd/vue-storefront

## Can you handle fixing this bug by yourself?

- [ ] YES
- [x] NO

## Which [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html) state this refers to? Info for developer.
Pick one option.

- [ ] This is a bug report for test version on https://test.storefrontcloud.io - In this case Developer should create branch from `develop` branch and create Pull Request `2. Feature / Improvement` back to `develop`.
- [ ] This is a bug report for current Release Candidate version on https://next.storefrontcloud.io - In this case Developer should create branch from `release` branch and create Pull Request `3. Stabilisation fix` back to `release`.
- [x] This is a bug report for current Stable version on https://demo.storefrontcloud.io and should be placed in next stable version hotfix - In this case Developer should create branch from `hotfix` or `master` branch and create Pull Request `4. Hotfix` back to `hotfix`.

## Environment details
- OS: Tested on OSX and with Travis
- Node: 8
- Code Version: e2c11b17cf44d761f268ae951685c11efc6bce7a

## Additional information
Error to be seen at: https://travis-ci.org/benjick/vsf-docker-problem/builds/541617124#L592

**Comment by pkarw** [comment]
could You heck it out @patzick please?

**Comment by indiebytes** [comment]
@benjick @patzick Looks like about the same issue as #3007 which was closed with a reference to pull request #3006. If my initial test is to be trusted, the same fix seem to resolve this issue as well.

**Comment by patzick** [comment]
Thanks @indiebytes! :) I will backport this with 1.9.2

**Comment by patzick** [comment]
Done, will be released with 1.9.2

**Comment by PawelGIX** [comment]
On latest master clean install of vue-storefront & docker-compose -f ./vue-storefront/docker-compose.yml up -d

```
plugin
<s> [webpack.Progress] 98% after emitting
<s> [webpack.Progress] 100% 

/var/www/core/lib/multistore.ts
./core/lib/multistore.ts
[tsl] ERROR in /var/www/core/lib/multistore.ts(153,43)
      TS2345: Argument of type '{ [key: string]: unknown; }' is not assignable to parameter of type 'Record<string, string | number | boolean | readonly (string | number | boolean)[]>'.
  Index signatures are incompatible.
    Type 'unknown' is not assignable to type 'string | number | boolean | readonly (string | number | boolean)[]'.
      Type '{}' is missing the following properties from type 'readonly (string | number | boolean)[]': length, concat, join, slice, and 16 more.
/var/www/core/lib/search/adapter/api/searchAdapter.ts
./core/lib/search/adapter/api/searchAdapter.ts
[tsl] ERROR in /var/www/core/lib/search/adapter/api/searchAdapter.ts(76,45)
      TS2345: Argument of type 'HttpQuery' is not assignable to parameter of type 'Record<string, string | number | boolean | readonly (string | number | boolean)[]>'.
  Index signature is missing in type 'HttpQuery'.
/var/www/core/lib/search/adapter/api-search-query/searchAdapter.ts
./core/lib/search/adapter/api-search-query/searchAdapter.ts
[tsl] ERROR in /var/www/core/lib/search/adapter/api-search-query/searchAdapter.ts(84,45)
      TS2345: Argument of type 'HttpQuery' is not assignable to parameter of type 'Record<string, string | number | boolean | readonly (string | number | boolean)[]>'.
<s> [webpack.Progress] 69% building 2701/2705 modules 4 active /var/www/node_modules/css-loader/index.js!/var/www/node_modules/vue-loader/lib/loaders/stylePostLoader.js!/var/www/node_modules/postcss-loader/src/index.js??postcss!/var/www/node_modules/sass-loader/dist/cjs.js!/var/www/node_modules/vue-loader/lib/index.js??vue-loader-options!/var/www/src/themes/capybara/pages/Static.vue?vue&type=style&index=0&id=5d2a47c7&lang=scss&scoped=true&
<s> [webpack.Progress] 69% building 2702/2705 modules 3 active /var/www/node_modules/css-loader/index.js!/var/www/node_modules/vue-loader/lib/loaders/stylePostLoader.js!/var/www/node_modules/postcss-loader/src/index.js??postcss!/var/www/node_modules/sass-loader/dist/cjs.js!/var/www/node_modules/vue-loader/lib/index.js??vue-loader-options!/var/www/src/themes/capybara/pages/Static.vue?vue&type=style&index=0&id=5d2a47c7&lang=scss&scoped=true&
<s> [webpack.Progress] 69% building 2703/2705 modules 2 active /var/www/node_modules/css-loader/index.js!/var/www/node_modules/vue-loader/lib/loaders/stylePostLoader.js!/var/www/node_modules/postcss-loader/src/index.js??postcss!/var/www/node_modules/sass-loader/dist/cjs.js!/var/www/node_modules/vue-loader/lib/index.js??vue-loader-options!/var/www/src/themes/capybara/pages/Static.vue?vue&type=style&index=0&id=5d2a47c7&lang=scss&scoped=true&
<s> [webpack.Progress] 69% building 2704/2705 modules 1 active /var/www/node_modules/css-loader/index.js!/var/www/node_modules/vue-loader/lib/loaders/stylePostLoader.js!/var/www/node_modules/postcss-loader/src/index.js??postcss!/var/www/node_modules/sass-loader/dist/cjs.js!/var/www/node_modules/vue-loader/lib/index.js??vue-loader-options!/var/www/src/themes/capybara/node_modules/@storefront-ui/vue/src/components/atoms/SfBreadcrumbs/SfBreadcrumbs.vue?vue&type=style&index=0&lang=scss&
<s> [webpack.Progress] 70% building 2705/2705 modules 0 active 
```
Do you have any idea what could be causing this error?
